### PR TITLE
DC Motor documentation

### PR DESCRIPTION
**Pull request by leandrumartin**
Added motor.adoc file. Currently it includes info about API calls, YAML file, and a link to the example controller.

### CODE REVIEW

File: pi4micronaut-utils/src/docs/asciidoc/components/outputComponents/motor.adoc
Comment (yrlmanoharreddy) [inline pi4micronaut-utils/src/docs/asciidoc/components/outputComponents/motor.adoc:1]: Remove "TODO: finish documentation here" under the Assembly Instructions.

**Review by yrlmanoharreddy — CHANGES_REQUESTED** [review]
The implementation aligns well with the project requirements, but please remove the "TODO: finish documentation here" note under the Assembly Instructions. From a product and code quality perspective, it's important to keep the code clean and avoid placeholder comments. Looking forward to the updates.

Todo:
Remove "TODO: finish documentation here" under the Assembly Instructions.

File: pi4micronaut-utils/src/docs/asciidoc/components/outputComponents/motor.adoc [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+Use the below commands to test the component.
+This will cause the motor to turn.
+
+[source,bash]
+----
+$ curl http://localhost:8080/motor/enable
+----
+
+* `/enable` - Enables the motor.
+* `/disable` - Disables the motor.
```
Comment (ThomasMacas) [inline pi4micronaut-utils/src/docs/asciidoc/components/outputComponents/motor.adoc:63]: Show how these could be run from command line.

**Review by yrlmanoharreddy — APPROVED** [review]
This pull request successfully implements the DC motor control functionality, providing comprehensive documentation that includes component descriptions, YAML configuration, circuit setup images, and clear usage instructions. The detailed information will greatly assist users in controlling motor speed and direction effectively.
Resolved #263

**Pull request merged**